docs/developers/meta/sc-meta-cli.md

```diff
@@ -209,6 +209,30 @@ Paramameter:
     -  The framework version on which the contracts should be created.
     - _default_: The latest released version.
 
+[comment]: # (mx-context-auto)
+
+### Calling `test`
+
+This command runs `cargo test`, which builds the project and runs the tests.
+
+Parameters:
+- `--path`  
+    - Target directory where to generate contract integration tests.
+    - _default_: current directory.
+- `--go`            
+    - This arg runs rust and go tests.
+    - _default_: `false`
+- `--scen`          
+    - This arg runs scenarios.
```

Comment:
This argument causes `cargo test` to be run with the [...] feature, causing tests relying on the mx-scenairo-go tool to also be run.